lib/localparticipant.js

```diff
@@ -371,6 +375,12 @@ class LocalParticipant extends Participant {
       return Promise.reject(error);
     }
 
+    const priorityValues = Object.values(trackPriority);
+    if (typeof options.priority !== 'string' || !priorityValues.includes(options.priority)) {
```

Comment:
ist part of the check is not necessary as `includes` cover it. 
```suggestion
    if (!priorityValues.includes(options.priority)) {
```